codice/ddf
# Ddf 4157 Added permissions and fixed bundles/features for profile-development install

#### What does this PR do?
This PR adds missing permissions for opendj and geowebcache which help aid in installing the profile-development. This also removes the geowebcache app from the dev install because geowebcache is off by default, and not recommended to be used in a production environment. A registry bundle was failing intermittently for the respective itest and was fixed as well. 

#### Who is reviewing it? 
@bakejeyner 
@ethantmanns 
@paouelle 
@jhunzik 

#### Select relevant component teams: 
@codice/test 


#### Ask 2 committers to review/merge the PR and tag them here.
@tbatie 
@figliold 

#### How should this be tested?
1. Install the profile-development feature, verify no issues before restarting 
2. Restart and verify that all bundles are working
3. Verify basic functionality still works
4. Run the ITInstallProfilesFeatures.java itests a few times and verify they pass

#### What are the relevant tickets?
[](https://codice.atlassian.net/browse/DDF-4157)


#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
